assets/helpers/internationalisation/countryGroup.js

```diff
@@ -0,0 +1,134 @@
+// @flow
+
+// ----- Imports ----- //
+
+import * as cookie from 'helpers/cookie';
+import { getQueryParameter } from 'helpers/url';
+
+import type { IsoCountry } from 'helpers/internationalisation/country';
+import type { IsoCurrency } from 'helpers/internationalisation/currency';
+
+export type CountryGroupId = 'GBPCountries' | 'UnitedStates' | 'AUDCountries' | 'EURCountries' | 'International';
+
+export type CountryGroup = {
+  name: string,
+  currency: IsoCurrency,
+  countries: IsoCountry[],
+  supportInternationalizationId: string,
```

Comment:
Can we have a comment explaining what this id is for?

Reply:
I think we should use the UK spelling (s not z) too so that we are the same as the Scala library. Will make github searching easier.